src/holofuel/UpdateHolofuelUserPrompt.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect, useCallback } from 'react'
+import { useHistory, Link } from 'react-router-dom'
+import { useQuery } from '@apollo/react-hooks'
+import { isEmpty } from 'lodash/fp'
+import HolofuelUserQuery from 'graphql/HolofuelUserQuery.gql'
+import useFlashMessageContext from 'holofuel/contexts/useFlashMessageContext'
+import { PROFILE_PATH } from 'holofuel/utils/urls'
+import 'holofuel/global-styles/colors.css'
+import 'holofuel/global-styles/index.css'
+
+function UserPromptMessage ({ newMessage }) {
+  const handleResetFlashMessage = useCallback(
```

Comment:
nitpicking (and not blocking), but we can just call this `resetFlashMessage`

`handle` implies this is a callback that is called WHEN the message is reset, rather than the function which is actually responsible for resetting the message,

Reply:
Good call. I support this thinking; updating it now.